projects/driver_controls/src/stm32f0xx/ads1015.c

```diff
@@ -173,7 +174,7 @@ StatusCode ads1015_read_converted(Ads1015Storage *storage, Ads1015Channel channe
   }
   int16_t raw_reading = ADS1015_READ_UNSUCCESSFUL;
   status_ok_or_return(ads1015_read_raw(storage, channel, &raw_reading));
-  if (raw_reading == ADS1015_DISABLED_CHANNEL_READING){
+  if (raw_reading == ADS1015_DISABLED_CHANNEL_READING) {
```

Comment:
Shouldn't `ads1015_read_raw` already return a non-OK status code if the channel is disabled?